### PR TITLE
✨ feature: restruct  docs

### PR DESCRIPTION
**Pull request by bingling-sama**
add file `basis.md`
provide some basic knowledge for zero-experience player to enhance their understanding of crash

### CODE REVIEW

**Comment by bingling-sama** [comment]
- [ ] a multi-level toc
- [ ] seperated and detailed instruction
- [ ] a footer displaying post author

**Comment by hejiehao** [comment]
We are Chinese. You can speak Chinese.

**Comment by bingling-sama** [comment]
> 

actually, it's my personal habits to use English on Github. if that's annoying u, I'll switch to Mandarin.

File: docs/basis.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+# 基础知识
+
+正在阅读本文档的玩家您好，我相信您此刻一定在游玩 `Minecraft` 时遇到了某些令人烦恼的问题。不论是单人愉悦游玩，或是架设服务器与好友畅玩。   
+
+> Minecraft这个游戏，各种BUG和解决方案如果写成一堆不重复的书，摞起来的厚度可以比姚明还高。（引用自[ DinnerBone 个人博客](https://www.cnblogs.com/Dinnerbone/p/12045117.html)）
```
Comment (Seayay) [inline docs/basis.md:5]: 请遵循 [中文文案排版指北](https://github.com/sparanoid/chinese-copywriting-guidelines/blob/master/README.zh-Hans.md)，在中英文之间增加空格

File: docs/basis.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+以下内容如果您有一定的计算机基础知识，那么您不必仔细地完整阅读，仅作为零基础读者加深对 `Minecraft` 游戏的理解的资料。
+:::
+
+## 启程之前
+
+在开始我们的 `Minecraft` 错误报告分析之前，我将先为您解开一部分心理上的疑惑。   
+
+> Q：我为什么要自己修崩溃？
+
+A：`Minecraft` 与您平时游玩的端游、页游、或是 `steam` 这种玩家平台上下载的游戏不同，它的初始定位是一款独立的单机游戏，再加上 `Minecraft` 特殊的开发技术栈，便出现了您需要自己去解决一些游戏错误甚至崩溃的情况。
```
Comment (Seayay) [inline docs/basis.md:19]: 请使用官方名称
steam --> Steam

File: docs/basis.md (hunk 1 above)
Comment (Big-Cake-jpg) [inline docs/basis.md:5]: 中英文字符间需要使用空格分隔。请参考中文文案排版指北。
```suggestion
> Minecraft 这个游戏，各种 Bug 和解决方案如果写成一堆不重复的书，摞起来的厚度可以比姚明还高。（引用自 [DinnerBone 个人博客](https://www.cnblogs.com/Dinnerbone/p/12045117.html)）
```

**Review by Seayay — CHANGES_REQUESTED** [review]

File: docs/basis.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+而由于开发 `Minecraft` 的工作室 `Mojang` 的一些原因，许多国内的玩家没办法使用 `正版启动器`（后文以 `官启` 代之）获得很好的游戏体验，因此便催生了一些社区开发者制作的 `第三方启动器`，此处列举几个国内著名的第三方启动器，如下表：
+
+|启动器名称|官网链接|开发者|
+|---|---|---|
+|HMCL|[Hello Minecraft! Launcher](https://hmcl.huangyuhui.net/)|huangyuhui|
+|BakaXL|[BakaXL 官方网站](https://www.bakaxl.com/)|TT|
+|PCL2|[龙猫爱发电](https://afdian.net/a/LTCat)|龙腾猫跃|
+|……|……|……|
+
+大部分时候， `Minecraft` 开发者们都会使用这些第三方启动器而不是使用官启，他们提供了更多的调试选项及错误输出为开发者解决问题提供方便。
```
Comment (Big-Cake-jpg) [inline docs/basis.md:42]: 全角标点符号后个人认为不应使用空格。

File: docs/basis.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+那么到了这一步，你已经找到了你游戏的崩溃日志，那么接下来你一定会想打开它看看你的 `Minecraft` 究竟发生了什么。而 `Minecraft` 的日志其实有很多种格式，常见的有 `.txt` 和 `.log`，而打开这两种格式的日志文件可以使用很多种编辑器，此处推荐几个好用的编辑器，如下：
+
+- [NotepadNext: A cross-platform, reimplementation of Notepad++.](https://github.com/dail8859/NotepadNext)
+:::details Q&A
+  > Q：为什么不用 Notepad++？   
+  > A：Notepad++ 的作者存在一些严重的政治立场问题，在这里不过多提及，但他本人将这种问题带入了他的作品之中，因此我本人并不推荐使用 Notepad++。
+:::
+- [Visual Studio Code: A source-code editor made by Microsoft](https://code.visualstudio.com)
+  > 一个轻量的、美观的文本编辑器，深受广大程序员喜爱。
+- Windows 记事簿（说实话 Windows10/11 的记事簿还是很好用的，如果你不想下载什么软件就用它吧）
```
Comment (Big-Cake-jpg) [inline docs/basis.md:66]: 使用官方称谓。
```suggestion
- Windows 记事本（说实话，Windows 10 / 11 的记事本还是很好用的，如果你不想下载什么软件就用它吧）
```

**Review by Big-Cake-jpg — CHANGES_REQUESTED** [review]
以及，行内代码与正文分隔只需使用一个空格即可。

File: docs/basis.md (hunk 3 above)
Comment (Seayay) [reply docs/basis.md:42]: > 全角标点符号后个人认为不应使用空格。

根据 [中文文案排版指北](https://github.com/sparanoid/chinese-copywriting-guidelines/blob/master/README.zh-Hans.md)，全角标点与其他字符之间不加空格。

详见[此处](https://github.com/sparanoid/chinese-copywriting-guidelines/blob/master/README.zh-Hans.md#%E5%85%A8%E8%A7%92%E6%A0%87%E7%82%B9%E4%B8%8E%E5%85%B6%E4%BB%96%E5%AD%97%E7%AC%A6%E4%B9%8B%E9%97%B4%E4%B8%8D%E5%8A%A0%E7%A9%BA%E6%A0%BC)

File: docs/basis.md [hunk 5]
```diff
@@ -63,7 +63,7 @@ A：要对 `Minecraft` 的游戏错误 / 崩溃进行处理，您首先要对游
 :::
 - [Visual Studio Code: A source-code editor made by Microsoft](https://code.visualstudio.com)
   > 一个轻量的、美观的文本编辑器，深受广大程序员喜爱。
-- Windows 记事簿（说实话 Windows10/11 的记事簿还是很好用的，如果你不想下载什么软件就用它吧）
+- Windows 记事簿（说实话, `Windows 10 / 11` 的记事簿还是很好用的，如果你不想下载什么软件就用它吧）
```
Comment (Seayay) [inline docs/basis.md:66]: 句中请使用全角符号

**Review by Seayay — CHANGES_REQUESTED** [review]

File: docs/basis.md (hunk 1 above)
Comment (bingling-sama) [reply docs/basis.md]: actually, that's a quote so i didnt notice it.

**Review by Seayay — APPROVED** [review]

File: docs/basis.md [hunk 6]
```diff
@@ -0,0 +1,70 @@
+# 基础知识
+
+正在阅读本文档的玩家您好，我相信您此刻一定在游玩 `Minecraft` 时遇到了某些令人烦恼的问题。不论是单人愉悦游玩，或是架设服务器与好友畅玩。   
+
+> `Minecraft` 这个游戏，各种BUG和解决方案如果写成一堆不重复的书，摞起来的厚度可以比姚明还高。（引用自[ DinnerBone 个人博客](https://www.cnblogs.com/Dinnerbone/p/12045117.html)）
```
Comment (Big-Cake-jpg) [inline docs/basis.md:5]:
```suggestion
> `Minecraft` 这个游戏，各种 Bug 和解决方案如果写成一堆不重复的书，摞起来的厚度可以比姚明还高。（引用自 [DinnerBone 个人博客](https://www.cnblogs.com/Dinnerbone/p/12045117.html)）
```

File: docs/basis.md [hunk 7]
```diff
@@ -0,0 +1,70 @@
...
+那么到了这一步，你已经找到了你游戏的崩溃日志，那么接下来你一定会想打开它看看你的 `Minecraft` 究竟发生了什么。而 `Minecraft` 的日志其实有很多种格式，常见的有 `.txt` 和 `.log`，而打开这两种格式的日志文件可以使用很多种编辑器，此处推荐几个好用的编辑器，如下：
+
+- [NotepadNext: A cross-platform, reimplementation of Notepad++.](https://github.com/dail8859/NotepadNext)
+:::details Q&A
+  > Q：为什么不用 Notepad++？   
+  > A：Notepad++ 的作者存在一些严重的政治立场问题，在这里不过多提及，但他本人将这种问题带入了他的作品之中，因此我本人并不推荐使用 Notepad++。
+:::
+- [Visual Studio Code: A source-code editor made by Microsoft](https://code.visualstudio.com)
+  > 一个轻量的、美观的文本编辑器，深受广大程序员喜爱。
+- Windows 记事簿（说实话， `Windows 10 / 11` 的记事簿还是很好用的，如果你不想下载什么软件就用它吧）
```
Comment (Big-Cake-jpg) [inline docs/basis.md:66]:
```suggestion
- Windows 记事本（说实话， `Windows 10 / 11` 的记事本还是很好用的，如果你不想下载什么软件就用它吧）
```
简体中文版 Windows 下显示为 “记事本”。

**Review by Big-Cake-jpg — CHANGES_REQUESTED** [review]

**Comment by bingling-sama** [comment]
> - [x] basic knowledge
> - [ ] a multi-level toc
> - [ ] seperated and detailed instruction
> - [x] ~~a footer displaying post author~~ (I'll move this to another pr)

about these two TODO:

I think in our future doc, the crash happens on `client` and `server` should be seperated into 2 sub dir. And we should refine the crash into more classes.

**Review by Big-Cake-jpg — DISMISSED** [review]

**Comment by bwtx2023** [comment]
Agree. 

However, I still want to notify that this doc will face pupils, but not only adults. So I hope that you can make the doc easier to read and understand. 

In #91, the analyzer is almost done, and you needn't put much time on it. Though, if you still have some enhancements, welcome to open an issue and discuss with us. 

(Off topic)

For convenient communication, I hope that you can join our QQ group. If you like, 199271393 is the group number. 

Thanks. 

---

同意上述观点。

但是，我仍然想要提醒，这篇文档不仅面向成人，也要面向小学生。所以我希望您能让文档更加易读并易于理解。

在 #91 中，分析器差不多完工了，您不用在它身上投入太多时间。尽管如此，如果您仍然有一些改进建议，欢迎开一个 Issue 并且和我们讨论。

（无关话题）

为了方便交流，我希望您还是能加入我们的 QQ 群。如果您想，群号是 199271393。

感谢。

**Comment by bingling-sama** [comment]
I see. I'll make it simpler and more detailed in the future.

File: docs/.vitepress/config.ts [hunk 8]
```diff
@@ -86,7 +86,15 @@ export default withPwa(
...
           items: [
             { text: "基础知识", link: "/basis" },
-            { text: "客户端崩溃", link: "/clients" },
+            {
+              text: "客户端崩溃",
+              collapsed: true,
+              items: [
+                { text: "系统问题", link: "/client/system" },
+                { text: "原版问题", link: "/client/vanilla" },
+                { text: "mod 问题", link: "/client/mods"},
```
Comment (Seayay) [inline docs/.vitepress/config.ts:95]: 大写 Mod

File: docs/client/mods.md [hunk 9]
```diff
@@ -0,0 +1,309 @@
+# 模组问题
+
+`Minecraft` 拥有庞大的玩家社区，也有许多的社区开发者制作 `mod` 为原版游戏增加扩展内容及玩法，但同时这也导致了许许多多的问题，这里列出部分常见的 `mod` 问题及解决方案。
```
Comment (Seayay) [inline docs/client/mods.md:3]: 大写

File: docs/client/mods.md [hunk 10]
```diff
@@ -0,0 +1,309 @@
...
+从官网下载并手动安装 Java: [Java 8](https://www.java.com/zh-CN/download/) | [JDK 17+](https://www.oracle.com/java/technologies/downloads/)
+
+获取 (Java 9+) 且已存档的 JDK 包: <https://jdk.java.net/archive/>
+:::
+
+## Mod 重复安装
+
+解释：由于有 Mod 重复安装，导致游戏无法正常运行。
+
+解决方案：排查重复的 Mod ，然后移除重复项直至只剩下一个该 Mod 文件。
```
Comment (Seayay) [inline docs/client/mods.md:77]: 全角符号与其他字符中间不添加空格

**Review by Seayay — CHANGES_REQUESTED** [review]

**Pull request merged**